src/main/java/hockeyapp/HockeyappRecorder.java

```diff
@@ -372,9 +372,11 @@ private boolean performForApplication(Run<?, ?> build, FilePath workspace, EnvVa
 
                     if (appIndex == 0) {
                         envData.add("HOCKEYAPP_CONFIG_URL", configUrl);
+                        logger.println("HOCKEYAPP_CONFIG_URL" + configUrl);
                     }
 
                     envData.add("HOCKEYAPP_CONFIG_URL_" + appIndex, configUrl);
+                    logger.println("HOCKEYAPP_CONFIG_URL_" + appIndex + ": " + configUrl);
```

Comment:
We can have multiple config (and install) urls returned from HockeyApp. As with the environment data object we expose all urls that we are able to. Do you still want to do that with your change in the console logger? If so you can keep this and the equivalent one for installation links below. Otherwise if you just want the first configuration url feel free to delete these lines....... but I suspect you probably want them too ;).